apps/src/aichat/views/AichatView.tsx

```diff
@@ -263,104 +270,111 @@ const AichatView: React.FunctionComponent<LabProps<AichatLevelProperties>> = ({
     );
   }, [dispatch]);
 
+  const showFlowLab = queryParams('show-flow-lab') === 'true' && isLevelbuilder;
```

Comment:
Could we do an early return here to avoid changes to the return statement below?

Reply:
Ah, of course.  Done.